pyproject.toml

```diff
@@ -20,7 +20,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.6"
-redis = "3.5.3"
+redis = "4.1.3"
```

Comment:
```
redis = "^4.1.3"
```

This keeps it within the same minor patch set - and we've already released 4.1.4

Reply:
Fixed